keystone.js

```diff
@@ -22,6 +22,8 @@ keystone.init({
 	'brand': 'SydJS',
 	'back': '/me',
 
+	'db': 'sydjs-site',
```

Comment:
is this something that was breaking for you? that should be the default database used, it's generated from the project name